main.js

```diff
@@ -3,46 +3,49 @@ saveBoxer.addEventListener('click', createBoxer);
 // const editBoxer = document.querySelector('#editBoxer');
 // editBoxer.addEventListener('click', editBoxer);
 
+
 function createBoxersArray() {
-  const newBoxer = [
-    firstName.value,
-    lastName.value,
-    nationality.value,
-    weightClass.value,
-  ]
-  localStorage.setItem('boxers', JSON.stringify(newBoxer));
-}
+  const boxers = [];
+  const newBoxer = {
+    firstName: firstName.value,
+    lastName: lastName.value,
+    nationality: nationality.value,
+    weightClass: weightClass.value,
+    wins: 0,
+    losses: 0
+  };
+  boxers.push(newBoxer);
+  localStorage.setItem('boxers', JSON.stringify(boxers));
```

Comment:
How is this not overriding the existing boxers each time you save it? 🤔

Reply:
It does overwrite them, but with the exact same information. Looking at this now, I think I should have done `localStorage.boxers = JSON.stringify(boxers);`